docs/multipass.md

```diff
@@ -0,0 +1,21 @@
+# Multipass
+[Shopify Multipass - Shopify Documentation](https://shopify.dev/docs/admin-api/rest/reference/plus/multipass) <br>
+This helper class generates token or URL that's needed for Shopify Multipass login (This feature is only available for Plus Store)
+```python
```

Comment:
I think it would be helpful to mention about enabling the multipass to obtain the secret. And that the URL generated with valid token is for redirecting the customer to the shopify store with and logged into the store.

Reply:
That is all in the Shopify doc. I think refer it to the official doc is better

Reply:
I'm with @ponty33 that it's best to let the Shopify docs speaks for itself and not duplicate. This way if it changes, we don't become out of date.